geomstats/geometry/positive_lower_triangular_matrices.py

```diff
@@ -0,0 +1,359 @@
+            dim=dim, signature=(dim, 0), default_point_type="matrix"
+        )
+        self.n = n
+
+    @staticmethod
+    def diag_inner_product(tangent_vec_a, tangent_vec_b, base_point):
+        """Compute the inner product using only diagonal elements.
+
+        Parameters
+        ----------
+        tangent_vec_a : array-like, shape=[..., n, n]
+            Tangent vector at base point.
+        tangent_vec_b : array-like, shape=[..., n, n]
+            Tangent vector at base point.
+        base_point : array-like, shape=[..., n, n]
+            Base point.
+
+        Returns
+        -------
+        ip_diagonal : array-like, shape=[...]
+            Inner-product.
+        """
+        inv_sqrt_diagonal = gs.power(Matrices.diagonal(base_point), -2)
+        tangent_vec_a_diagonal = Matrices.diagonal(tangent_vec_a)
+        tangent_vec_b_diagonal = Matrices.diagonal(tangent_vec_b)
+        prod = tangent_vec_a_diagonal * tangent_vec_b_diagonal * inv_sqrt_diagonal
+        ip_diagonal = gs.sum(prod, axis=-1)
+        return ip_diagonal
+
+    @staticmethod
+    def strictly_lower_inner_product(tangent_vec_a, tangent_vec_b):
+        """Compute the inner product using only strictly lower triangular elements.
+
+        Parameters
+        ----------
+        tangent_vec_a : array-like, shape=[..., n, n]
+            Tangent vector at base point.
+        tangent_vec_b : array-like, shape=[..., n, n]
+            Tangent vector at base point.
+
+        Returns
+        -------
+        ip_sl : array-like, shape=[...]
+            Inner-product.
+        """
+        sl_tagnet_vec_a = gs.tril_to_vec(tangent_vec_a, k=-1)
+        sl_tagnet_vec_b = gs.tril_to_vec(tangent_vec_b, k=-1)
+        ip_sl = gs.einsum("...i,...i->...", sl_tagnet_vec_a, sl_tagnet_vec_b)
+        return ip_sl
+
+    @classmethod
+    def inner_product(cls, tangent_vec_a, tangent_vec_b, base_point):
+        """Compute the inner product using only strictly lower triangular elements.
+
+
+        Compute the inner-product of tangent_vec_a and tangent_vec_b
+        at point base_point using the cholesky Riemannian metric.
+
+        Parameters
+        ----------
+        tangent_vec_a : array-like, shape=[..., n, n]
+            Tangent vector at base point.
+        tangent_vec_b : array-like, shape=[..., n, n]
+            Tangent vector at base point.
+        base_point : array-like, shape=[..., n, n]
+            Base point.
+
+        Returns
+        -------
+        inner_product : array-like, shape=[...]
+            Inner-product.
+        """
+        diag_inner_product = cls.diag_inner_product(
+            tangent_vec_a, tangent_vec_b, base_point
+        )
+        strictly_lower_inner_product = cls.strictly_lower_inner_product(
+            tangent_vec_a, tangent_vec_b
+        )
+        return diag_inner_product + strictly_lower_inner_product
+
+    def exp(self, tangent_vec, base_point, **kwargs):
+        """Compute the Cholesky exponential map.
+
```

Comment:
Could you put the equation in :math: in the docstring? (I see it is in the paper), while referencing the paper again? Thank you!